### PR TITLE
Wish list Page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

File: mycart.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.mycart{
+    min-height: 50vh;
+}
+.mycart-container{
+    background-color: tomato;
+    height: 40px;
+
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-width: 1400px;
```
Comment (Sahil25061999) [inline mycart.css:10]: Just a suggestion use rem here.

File: wishlist.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.wishlist-products{
+    min-height: 50vh;
+}
+.wishlist-products-container{
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    gap: 2rem;
+    margin: 0rem 2rem;
```
Comment (sri-rishi) [inline wishlist.css:9]: just a suggestion use 0 instead of 0rem

**Review by sri-rishi — COMMENTED** [review]
Looks great to merge

File: mycart.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+      <div class="spacer-3"></div>
+      <div class="mycart-container">
+        <div class="mycart-items-container">
+          <div class="card card-horizontal">
+            <div class="image-container">
+              <img
+                alt="Reebok Resistant Tube"
+                src="https://cdn.shopify.com/s/files/1/0610/2882/7370/products/product38.jpg?v=1636690872"
+                class="img-responsive"
+              />
```
Comment (gauravsinhaweb) [inline mycart.html:93]: good practice is to use class first then src then alt.

File: mycart.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        </div>
+      </div>
+      <div class="nav-items-container-end">
+        <div class="login-search-bar-container">
+          <button class="button button-primary">Login</button>
+          <div class="input-box">
+            <input
+              type="text"
+              class="input"
+              value=""
```
Comment (gauravsinhaweb) [inline mycart.html:58]: give value element some value.

File: mycart.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+  <body>
+    <!--  ******************************** Navbar Start ********************************  -->
+    <nav class="navbar">
+      <div>
+        <img
+          alt="hamburger-icon"
+          src="images/hamburger.svg"
+          class="hamburger-icon"
+          id="hamburger-icon"
```
Comment (gauravsinhaweb) [inline mycart.html:41]: keep elements in sequence

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: mycart.html (hunk 4 above)
Comment (Ankur9669) [reply mycart.html:58]: Actually value should be emoty in the begining as there is nothing in the input

**Pull request merged**